apps/condo/domains/user/access/User.js

```diff
@@ -184,7 +172,33 @@ async function canReadUserNameField (args) {
 const canAccessMarketingConsent = {
     read: true,
     create: access.userIsAdminOrIsSupport,
-    update: access.userIsAdminOrIsThisItem,
+    update: userIsAdminOrCanDirectlyManageField,
+}
+
+async function userIsAdminOrIsThisItemOrCanDirectlyReadField (args) {
+    const nonDirectAccess = access.userIsAdminOrIsThisItem(args)
+
+    if (nonDirectAccess) {
+        return nonDirectAccess
+    }
+
+    const { authentication: { item: user }, listKey, fieldKey } = args
+
+    return await canDirectlyReadSchemaField(user, listKey, fieldKey)
+}
+
+async function userIsAdminOrCanDirectlyManageField ({ authentication: { item: user }, listKey, fieldKey }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    return await canDirectlyManageSchemaField(user, listKey, fieldKey)
+}
+
+const canAccessRightsSet = {
+    read: access.userIsAdminOrIsSupport,
```

Comment:
What if have direct access to UserRightsSet model? Or to user.rightsSet field? Why only support and admin can read it? Before it was true?